style-login.css

```diff
@@ -28,6 +28,8 @@ body.login h1 a {
 body.login div#login form#loginform p.galogin a {
   text-decoration: none;
   font-weight: normal;
+  display: block;
+  line-height: 3rem;
```

Comment:
Since you changed the display to block, you can just add some padding instead of changing the line-height. A 5px padding feels close to the [required design](https://jira.greenpeace.org/secure/attachment/27966/login_adjusted.jpg).

Reply:
![image](https://user-images.githubusercontent.com/2221076/58998029-3fbbc480-8842-11e9-8ec0-4238f9559ee2.png)
Without modifying line-height, and adding 5px, the result is unbalanced. I've modified the line-height to 2rem to bring it into alignment with the design guide.